CHANGELOG.md

```diff
@@ -13,3 +13,16 @@ expect('google.com').to redirect_http_to_https.then be_successful
 
 This enables more fine-grained checks but also means you'll need to revise your tests when you update.
 
+## Caching
+
+We had been checking the `X-Check-Cacheable` header.
+As Akamai no longer recommends relying on this header, we now make
+a sequence of requests and look for `X-Cache: TCP_HIT`.
```

Comment:
Maybe be a bit more explicit, ie. if _any_ response from the sequence of hits is cached then this matcher will pass.